MediaBrowser.WebDashboard/Api/PackageCreator.cs

```diff
@@ -44,10 +47,6 @@ public static bool IsCoreHtml(string path)
             return string.Equals(Path.GetExtension(path), ".html", StringComparison.OrdinalIgnoreCase);
         }
 
-        /// <summary>
-        /// Modifies the HTML by adding common meta tags, css and js.
-        /// </summary>
-        /// <returns>Task{Stream}.</returns>
```

Comment:
why remove?

Reply:
Analyzer wants full documentation

Reply:
But that really describes what it does, maybe we can at least keep a `//` comment right before the method?